javascript-data-model/order-history.js

```diff
@@ -0,0 +1,79 @@
+var orderHistory = [
+  {
+    orderNumber: '113-3941689-8772232',
+    dateOrderPlaced: 'August 4, 2020',
+    totalBeforeTax: '$31.55',
+    totalAfterTax: '$34.00',
+    grandTotal: '$34.00',
+    dateDelivered: 'Aug 8, 2020',
+    shipTo: 'JS Masher',
+    returnWindowCloseDate: 'September 7th, 2020',
+    returnWindowClosed: true,
+    packageDelivered: true,
+    handedToResident: true,
+    productCategory: 'books',
+    title: 'Javascript for Impatient programmers',
+    author: 'Rauschmayer, Dr. Axel'
+  },
+  {
+    orderNumber: '113-9984268-1280257',
+    dateOrderPlaced: 'July 19, 2020',
+    totalBeforeTax: '$41.33',
+    totalAfterTax: '$44.53',
+    grandTotal: '$44.53',
+    dateDelivered: 'Jul 20, 2020',
+    shipTo: 'JS Masher',
+    returnWindowCloseDate: 'Aug 19, 2020',
+    returnWindowClosed: true,
+    packageDelivered: true,
+    handedToResident: true,
+    productCategory: 'books',
+    title: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher'
+  },
+  {
+    orderNumber: '114-2875557-9059409',
+    dateOrderPlaced: 'July 4, 2020',
+    totalBeforeTax: '$15.98',
+    totalAfterTax: '$17.22',
+    grandTotal: '$17.22',
+    dateDelivered: 'Jul 7, 2020',
+    shipTo: 'JS Masher',
+    returnWindowCloseDate: 'Aug 5 2020',
+    returnWindowClosed: true,
+    productDelvered: true,
+    handedToResident: true,
+    productCategory: 'electronics',
+    title: 'Gamecube Controller Adapter, Super Smash Bros Switch Gamecube Adapter for Wii U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter'
+  },
+  [{
+    orderNumber: '113-2883177-2648248',
+    dateOrderPlaced: 'July 3, 2020',
+    totalBeforeTax: '$94.95',
+    totalAfterTax: '$138.93',
+    grandTotal: '$138.93',
+    dateDelivered: 'Jul 5, 2020',
+    shipTo: 'JS Masher',
+    returnWindowCloseDate: 'Aug 4, 2020',
+    returnWindowClosed: true,
+    handedToResident: false,
+    productCategory: 'electronics',
+    title: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)'
+  },
+
+  {
+    orderNumber: '113-2883177-2648248',
+    dateOrderPlaced: 'July 3, 2020',
+    totalBeforeTax: '$33.99',
+    totalAfterTax: '$138.93',
+    grandTotal: '$138.93',
+    dateDelivered: 'Jul 5, 2020',
+    shipTo: 'JS Masher',
+    returnWindowCloseDate: 'Aug 4, 2020',
+    returnWindowClosed: true,
+    handedToResident: false,
+    productCategory: 'books',
+    title: 'The Art of Sql',
+    author: 'Faroult, Stephane'
+  }]
```

Comment:
So this order is an array, but each of your other orders is an object. That makes this data model inconsistent, which would lead to errors.  You should have one object for the order, and use an array for just the items in the order.